Source/Foundation/bsfUtility/Prerequisites/BsStdHeaders.h

```diff
@@ -223,12 +223,12 @@ namespace bs
 		constexpr T& operator*() const { return *mPtr; }
 		constexpr T* operator->() const { return mPtr; }
 		constexpr T* get() const { return mPtr; }
-		constexpr bool operator< (const NativePtr& rhs) const { return mPtr <  rhs.mPtr; }
-		constexpr bool operator> (const NativePtr& rhs) const { return mPtr >  rhs.mPtr; }
-		constexpr bool operator<=(const NativePtr& rhs) const { return mPtr <= rhs.mPtr; }
-		constexpr bool operator>=(const NativePtr& rhs) const { return mPtr >= rhs.mPtr; }
-		constexpr bool operator==(const NativePtr& rhs) const { return mPtr == rhs.mPtr; }
-		constexpr bool operator!=(const NativePtr& rhs) const { return mPtr != rhs.mPtr; }
+		bool operator< (const NativePtr& rhs) const { return mPtr <  rhs.mPtr; }
```

Comment:
Hold on. Why did you need to remove the constexpr?

Reply:
That's the unfortunate consequence of having to specialize the template, since specialization had to be calling non-constexpr functions, it couldn't be constexpr itself :disappointed: